RationaleMCP/0031/differences.md

```diff
@@ -296,8 +334,10 @@ end 'fun';
 ```
 
 The following restriction applies to modifications in types and functions, making types and function signatures in Flat Modelica easier to represent and reason about compared to full Modelica:
-- Modifiers must have constant variability.
-- Modifiers must be scalar, giving all elements of an array the same element type.  Details of how the scalar modifier is applied to all elements of an array is described [below](#Single-array-element-type).  For example, an array in a type cannot have individual element types with different `unit` attributes.
+- Attribute modifiers must have constant variability.
```

Comment:
Add explicitly that value modifications in model component declarations are not affected.

Reply:
This is already stated in the proper place; this list is only about types and functions.

Reply:
Resolved in web meeting